### DISCUSSION TITLE
Can something be done to add rounded corners using mutter-rounded

### DISCUSSION THREAD

**Discussion by xAspirus** [rafaelmardojai/firefox-gnome-theme#450] [Q&A] [answered]
If using mutter-rounded on wayland

**Comment by CleoMenezesJr** [comment, marked as answer]
I prefer to recommend using the [Rounded Window Corners](https://extensions.gnome.org/extension/5237/rounded-window-corners/) extension instead of installing a patch on Mutter. It is by far the best solution. This extension is made by the same author of mutter-rounded.
![image](https://user-images.githubusercontent.com/54215258/202862387-8684ac2d-1b8b-480f-8ef7-9532fc750ac3.png)
